DS_and_Algorithm/Day3/Selectionsort.py

```diff
@@ -0,0 +1,40 @@
+def selection_sort(arr):
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/11#issuecomment-2244955315>#03633d</a><span> - 07/23/2024, 11:19 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The function 'selection_sort' does not validate the input type. If a non-list input is provided, it could lead to unexpected behavior or runtime errors.
</div>
<div id="fix">
<b>Fix:</b> Add input validation to ensure that the provided argument is a list.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,3 +1,6 @@
 def selection_sort(arr):
     """
     Sorts an array using the Selection Sort algorithm.
 +    
 +    if not isinstance(arr, list):
 +        raise TypeError("Input must be a list")
    
     Parameters:
     - arr (list): The input list to be sorted in-place.
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged